docs/download.md

```diff
@@ -43,7 +43,7 @@ The plugin is tested on the following architectures:
 		Apache Spark 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.2.4
 		Apache Spark 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4
 		Apache Spark 3.4.0, 3.4.1, 3.4.2, 3.4.3
```

Comment:
We added support for 3.4.4 as well in https://github.com/NVIDIA/spark-rapids/pull/11692
```suggestion
		Apache Spark 3.4.0, 3.4.1, 3.4.2, 3.4.3, 3.4.4
```